### PR TITLE
Initial commit for the Italian translation

### PR DESCRIPTION
**Pull request by Aury88**
- Create the res/locales/it and added the italian translation for intent and messages and started modules

### CODE REVIEW

**Comment by hugolgst** [comment]
Thanks for that!
Did you copied the `training.json` file or re-trained the model ?

File: res/locales/it/modules.go [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+	// COUNTRIES
+	// Please translate this method for adding the correct article in front of countries names.
+	// Otherwise, remove the countries modules from this file.
+
+	modules.ArticleCountries["it"] = ArticleCountries
+}
+
+// ArticleCountries returns the country with its article in front.
+func ArticleCountries(name string) string {
```
Comment (hugolgst) [inline res/locales/it/modules.go:233]: You should either create a method that finds the article of a country or remove it

Comment (Aury88) [reply res/locales/it/modules.go:233]: that file is  a WIP. I have copied the en training.json and translated it

Comment (hugolgst) [reply res/locales/it/modules.go:234]: Okay, good luck!

File: res/locales/it/stopwords.txt [hunk 2]
```diff
@@ -0,0 +1,98 @@
+a
```
Comment (hugolgst) [inline res/locales/it/stopwords.txt:1]: You need to put an Italian list of stopwords, you can search on this repository:
https://github.com/stopwords-iso/stopwords-it

**Comment by Aury88** [comment]
added the method that finds the article of a country...I don't know how that is applied if only on the answer or also in the request...in Italia the request for the surface has the same article applied in the answer...foe exaple with the surface the question should be: "Qual è la superficie **degli** Stati Uniti "(for US),  "Qual è la superficie **della** Francia" (for fr)  and "Qual è la superficie **dell'**Italia (for it)

**Comment by Aury88** [comment]
thank you @hugolgst for the improvement in the modules.go. I'm not a developer so your change was really instructive.
For the stopword I was trying to understand if I have to put all the words listed in the https://github.com/stopwords-iso/stopwords-it or if I must use only some of them. The documentation says to translate the strings and that put me a little bit off track. so thank you also for that.
The only things I don't understand is why you de-commented the code in eg. the math module in the modules.go...the math module comment says that a regex translation is required in `language/math.go since I didn't translate that files yet ( and I don't know when I will be able to do that) I thought the module should have been removed as requested by the comment.

**Comment by hugolgst** [comment]
Oh I'm sorry I did that, you're right I thought you were adding it sorry for that

**Comment by hugolgst** [comment]
Thanks for this translation!
Could you also try to translate the frontend? Here is the documentation part for it: https://docs.olivia-ai.org/translations.html#frontend

**Pull request merged**